mock-server-frontend/src/components/EndpointTable.js

```diff
@@ -49,40 +47,49 @@ export default function EndpointTable() {
             .then(res => {
                 let data = res.data.map(x => {
                     x.isSaved = true
-                    x.value = JSON.stringify(x.value)
+                    x.value = JSON.stringify(x.value, null, 2)
```

Comment:
are there anyway to use named parameters? so we can easily understand what are those magic values?